### PR TITLE
Add a Python-specific landing page

### PR DESCRIPTION
**Pull request by ericholscher**
This reuses a lot of content but also tries a few new approaches.

## Preview

127 0 0 1_8000_publishers_python_

### CODE REVIEW

File: content/pages/landing-pages/python.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: Privacy-first ads on Python projects
+slug: python
+url: publishers/python/
```
Comment (davidfischer) [inline content/pages/landing-pages/python.md:3]: Just a thought: Should it be `/publishers/topics/python/`?

File: ethicalads-theme/templates/ea/landing-pages/python.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+<!-- Sample Ad -->
+<!-- ================================================== -->
+<section class="py-10">
+  <div class="container">
+    <div class="row justify-content-center">
+
+      <div class="col-md-6 mb-3">
+        <h2 class="font-weight-bold mb-8">Python-focused ads</h2>
+        <p class="font-size-lg text-gray-800">We use the content of your pages to target advertising, so you'll only get ads that are relevent to your content. <strong>No ads that track your users around the internet.</strong> You can see every advertiser that shows an ad on your site.</p>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/landing-pages/python.html:108]: Technically they only see the top 20 but I guess it's a fine claim.

**Review by davidfischer — APPROVED** [review]
This looks like a good start. Where should we link to this from? It would be nice to ensure it's indexed.

File: ethicalads-theme/templates/ea/landing-pages/python.html (hunk 2 above)
Comment (ericholscher) [reply ethicalads-theme/templates/ea/landing-pages/python.html]: Said `major` 👍

**Comment by ericholscher** [comment]
@davidfischer addressed feedback. Also rejiggered the footer to make it a bit more balanced:

Screen Shot 2021-08-02 at 4 11 14 PM

**Comment by davidfischer** [comment]
:ship: Ship it!

**Pull request merged**